retrofit/src/test/java/retrofit/RequestBuilderTest.java

```diff
@@ -1008,22 +1009,22 @@ Response method(
     Request request = buildRequest(Example.class, params);
     assertThat(request.method()).isEqualTo("GET");
     assertThat(request.headers().size()).isZero();
-    assertThat(request.urlString()).isEqualTo("http://example.com/foo/bar/?k+it=k%20t&pi+ng=p%20g");
+    assertThat(request.urlString()).isEqualTo("http://example.com/foo/bar/?k%20it=k%20t&pi%20ng=p%20g");
     assertThat(request.body()).isNull();
   }
 
   @Test public void normalPostWithPathParam() {
     class Example {
       @POST("/foo/bar/{ping}/") //
-      Response method(@Path("ping") String ping) {
+      Response method(@Path("ping") String ping, @Body Object body) {
```

Comment:
okhttp 2.3 allows to pass `null` when `RequestBody` is constructed, which is done by retrofit 1.9.
With okhttp 2.4 an empty body must be defined explicitly (e.g. with an empty array).
This change removes a test for post requests with empty body. Will they be supported?